meson.build

```diff
@@ -298,12 +298,12 @@ data_files = files(
 	'data/devices/logitech-g700s.device',
 	'data/devices/logitech-g703.device',
 	'data/devices/logitech-g9.device',
-	'data/devices/logitech-g900-wireless.device',
 	'data/devices/logitech-g900.device',
 	'data/devices/logitech-g903.device',
 	'data/devices/logitech-g9x-[Call-of-Duty-MW3-Edition].device',
 	'data/devices/logitech-g9x-[Original].device',
 	'data/devices/logitech-gpowerplay-wireless.device',
+	'data/devices/logitech-lightspeed.device',
```

Comment:
can't comment on the renamed file but the `Name=` should be changed to `Logitech device connected via Lightspeed Connector` or so.

Reply:
Right, totally forgot.